### PR TITLE
Add Lessons button

### PR DESCRIPTION
**Pull request by sweisz**
- add 2 buttons for lessens page to change the Skill level.

### CODE REVIEW

File: components/LessonsButton/LessonsButton.module.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.btn {
+  border: none;
+  background-color: #959595;
+  color: #f2f2f2;
+  padding: 1em;
+  margin-right: 1px;
```
Comment (mewiemers) [inline components/LessonsButton/LessonsButton.module.css]: can you better use relative units?

**Review by mewiemers — COMMENTED** [review]
Good Job! 🎸🎶

**Review by SilviaBarcellini — APPROVED** [review]
lgtm 👍

File: components/LessonsButton/LessonsButton.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import styles from "./LessonsButton.module.css";
+import { useRouter } from "next/router";
+
+export type LessonsButtonProps = {
+  labelLeft: string;
+  labelRight: string;
+  route: string;
```
Comment (lmachens) [inline components/LessonsButton/LessonsButton.tsx]: Wouldn't it make more sense to have two routes, one for each link?

File: components/LessonsButton/LessonsButton.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+};
+
+function LessonsButton({ labelLeft, labelRight, route }: LessonsButtonProps) {
+  const router = useRouter();
+  return (
+    <div className={styles.container}>
+      <button className={styles.btn} onClick={() => router.push(route)}>
```
Comment (lmachens) [inline components/LessonsButton/LessonsButton.tsx]: If this button is only for navigation, use `a` or [`Link`](https://nextjs.org/docs/api-reference/next/link) instead. Links are for navigation, buttons a for general actions.

You will find good articles about this topic by searching for `anchor vs button`

File: components/LessonsButton/LessonsButton.module.css [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+}
+.btn:last-child {
+  margin-right: 0;
```
Comment (lmachens) [inline components/LessonsButton/LessonsButton.module.css]: This works well, but there is a good alternative to achieve a similar gap between your buttons without any `margin`.
You can add `grid-column-gap` to your container and remove all margins.

**Review by lmachens — COMMENTED** [review]
Good job

File: components/LessonsButton/LessonsButton.module.css (hunk 1 above)
Comment (sweisz) [reply components/LessonsButton/LessonsButton.module.css]: Fixed with column-gap

**Pull request merged**